csharp_dotnetcore/81.Facebook-Events/Facebook-Events-Bot.bot

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Facebook-Events-Bot",
+    "description": "",
+    "secretKey": "",
+    "services": [
+      {
+        "type": "endpoint",
+        "name": "facebook-events",
+        "id": "https://2137f71a.ngrok.io/api/messages",
+        "appId": "1999c181-69b1-4705-a51d-cd3071e8532d",
+        "appPassword": "mgmgkKKY7664!lpSMMN2:]_",
+        "endpoint": "https://2137f71a.ngrok.io/api/messages"
```

Comment:
Remove the AppID and Password. 

... and now that they're in Git History, rotate the keys.